python/tests/scripts/d2_higherfunc.metta

```diff
@@ -168,12 +168,14 @@
 !(assertEqualToResult
   (get-type (fmap (curry-a + 2) (Left "5")))
   ())
-!(assertEqualToResult
-  (get-type (fmap (curry-a + 2) (UntypedC "5")))
-  ())
-!(assertEqualToResult
-  (get-type (fmap (curry-a + 2) (UntypedC (Null) 5)))
-  ())
+; Two examples below are type-checked successfully because, (UntypedC "5")
```

Comment:
Yes, I agree. The type of `(UntypedC "5")` should be reducible to `%Undefined%` meaning that the whole expression can type-check. I'd add "FIXME" or "TODO", because we may need to revisit these tests and uncomment their updated versions